src/components/SideBar.astro

```diff
@@ -88,3 +88,31 @@
     </div>
   </div>
 </div>
+<script>
+  const homeLink = document.getElementById('home');
+  const projectsLink = document.getElementById('projects');
+  const servicesLink = document.getElementById('services');
+  const storeLink = document.getElementById('store');
+  const blogLink = document.getElementById('blog');
+  const cvLink = document.getElementById('cv');
+
+  var isHome = true;
+
+  const url = window.location.href;
+
+  const pages = [ projectsLink, servicesLink, storeLink, blogLink, cvLink ];
+
+  pages.forEach( (elem) => {
+    if (url.includes(elem.id)) {
+      isHome = false;
+      elem.classList.add('bg-neutral');
```

Comment:
In daisy UI they use a class `active` for the list items, so we can use that here.